web_company_color/view/res_company.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+
+<!--
+    Copyright 2019
+    @author Alexandre Díaz <dev@redneboa.es>
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+
+    <record id="view_company_form" model="ir.ui.view">
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form" />
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Navbar" name="navbar" groups="base.group_system">
```

Comment:
pls rename the tab to "Company styles" or "Backend styles" or similar.
We might add some other styles not related to navbar only.

Reply:
Done